java/java.source.base/src/org/netbeans/modules/java/source/TreeShims.java

```diff
@@ -50,6 +55,63 @@ public static Tree getBody(CaseTree node) {
         }
     }
 
+    public static List<? extends ExpressionTree> getExpressions(Tree node) {
+        List<? extends ExpressionTree> exprTrees = new ArrayList<>();
+
+        switch (node.getKind().toString()) {
+            case "CASE":
+                exprTrees = getExpressions((CaseTree) node);
+                break;
+            case "SWITCH_EXPRESSION": {
+                try {
+                    Class swExprTreeClass = Class.forName("com.sun.source.tree.SwitchExpressionTree");
```

Comment:
Please use "// NOI18N" comment for hard coded strings

Reply:
TreeShims is planned to be removed in future , so i believe // NOI18N" is not required